app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,214 @@
  *= require_tree .
  *= require_self
  */
+
+.navbar {
```

Comment:
Good use of CSS, namings are clear therefore make it easy to change in the future

Reply:
I have a good CSS Flexbox tutorial to quickly learn and apply it in practice